btrack/btypes.py

```diff
@@ -257,7 +261,7 @@ class Tracklet:
         The objects linked together to form the track.
     parent : int,
         The identifiers of the parent track(s).
-    children : list
+    children : list or None
```

Comment:
I still think we shouldn't have types in docstrings

Reply:
I added it to be consistent with other parts of the code I've seen. Happy to remove it though

Reply:
More just raising the point/making you aware

Reply:
I've removed the `or None` from the docstring